### ISSUE TITLE
XSD is downloaded and extracted for every build on OSX

### ISSUE THREAD

**Issue by theold190** [percolator/percolator#65]
Every time I start osx_build.sh script xsd is downloaded and extracted to the temporary folder. It is never installed into the system.

Would be nice to have xsd downloaded and extracted only once. And always be used from a predefined folder (ex. percolator/build/tools).

**Comment by MatthewThe** [comment]
Fixed in 61e10b04c295888ee5579816957262043e00266b
